references/organization/_06-Project-Plan-and-Workflow.md

```diff
@@ -0,0 +1,64 @@
+-	In the second part, we use the same variables, but now we simulate the data based on a more realistic causal graph where some variables influence/cause the others. We then simulate choice data based on the same model used in part 1. Finally, we sample again from this population multiple times and re-estimate the outcome model, and look at the percentage of time that we recover the true parameters on our variables of interest. What we should find here is that asymptotically, we never recover the true parameters on any variables that have descendants in the causal graph that we control for. <br>
+The conclusion from this is that even though the outcome model is the same for the data generation process, we can only assign causal interpretations to parameters if we have clearly drawn our assumptions on the data generating process. Without those, one may easily challenge any interpretation we assign to our parameters by simply asking the question “but what if the data generating process looked like this?”. By drawing a causal graph upfront, we limit this sort of questions to questions about the validity of one’s causal graph, which comes with testable implications and create much more interesting and focused discussions than those created when there’s no causal graph that clearly shows the modeler’s assumptions. 
+
+##### Concrete steps:
+
+Functions <br>
+-	The main function we need is one that takes as input a hypothesized causal graph (through a set of nodes and edges), with distributional assumptions and parametric relationships between variables, and simulates data given those inputs. 
+-	Since we’re dealing with a multinomial choice problem, we need a function to simulate outcomes and be flexible to take in multiple specifications for each utility equation.
+
+Workflow <br>
+-	Come up with an outcome (choice) model based on the bike data in Brathwaite (2018). The model doesn’t need to be the exact model estimated in the paper, but that can be the case. We just need to select a set of variables while being mindful that one of those variables will be used as a latent confounder in the next step. 
+-	Simulate two sets of data for the X variables: <br>
+    - One where all the Xs are independent (i.e. the only edges in the causal graph are between the X and the outcome variable). 
+    - One based on a realistic causal graph with confounders. Preferably focus on a case with one latent confounder (like distance? Age? Gender?) <br>
+-	For each of the two causal graphs above, simulate choice data based on the outcome model assumed. <br>
+-	Estimate the choice model for each of those two datasets. <br>
```

Comment:
Estimate the choice model for each of the two datasets using the assumed choice model specification.